lightwood/encoder/text/pretrained.py

```diff
@@ -135,35 +80,43 @@ def __init__(
         self.uses_target = True
         self.output_size = None
 
-        # DEBUGGING!!!
         if self.embed_mode:
             log.info("Embedding mode on. [CLS] embedding dim output of encode()")
         else:
             log.info("Embedding mode off. Logits are output of encode()")
 
-    def prepare(self, train_priming_data: pd.Series, dev_priming_data: pd.Series, encoded_target_values: torch.Tensor):
+    def prepare(
+        self,
+        train_priming_data: pd.Series,
+        dev_priming_data: Optional[pd.Series],
```

Comment:
make these `Iterable` instead of pd series.

Reply:
sounds good